core/dungeon-masters-guide/items/items-weapons.xml

```diff
@@ -538,6 +538,199 @@
+			<p class="indent"><i><b>Random Properties.</b></i> The axe has the following randomly determined properties:</p>
+			<ul>
+				<li>2 minor beneficial properties</li>
+				<li>1 major beneficial property</li>
+				<li>2 minor detrimental properties</li>
+			</ul>
+			<p class="indent"><i><b>Blessings of Moradin.</b></i> If you are a dwarf attuned to the axe, you gain the following benefits:</p>
+			<ul>
+				<li>You have immunity to poison damage.</li>
+				<li>The range of your darkvision increases by 60 feet.</li>
+				<li>You gain proficiency with artisan's tools related to blacksmithing, brewing, and stonemasonry.</li>
+			</ul>
+			<p class="indent"><i><b>Conjure Earth Elemental.</b></i> If you are holding the axe, you can use your action to cast the <i>conjure elemental</i> spell from it, summoning an earth elemental. You can't use this property again until the next dawn.</p>
+			<p class="indent"><i><b>Travel the Depths.</b></i> You can use an action to touch the axe to a fixed piece of dwarven stonework and cast the <i>teleport</i> spell from the axe. If your intended destination is underground, there is no chance of a mishap or arriving somewhere unexpected. You can't use this property again until 3 days have passed.</p>
+			<p class="indent"><i><b>Curse.</b></i> The axe bears a curse that affects any non-dwarf that becomes attuned to it. Even if the attunement ends, the curse remains. With each passing day, the creature's physical appearance and stature become more dwarflike. After seven days, the creature looks like a typical dwarf, but the creature neither loses its racial traits nor gains the racial traits of a dwarf. The physical changes wrought by the axe aren't considered magical in nature (and therefore can't be dispelled), but they can be undone by any effect that removes a curse, such as a <i>greater restoration</i> or <i>remove curse</i> spell.</p>
+			<p class="indent"><i><b>Destroying the Axe.</b></i> The only way to destroy the axe is to melt it down in the <em>Earthheart Forge</em>, where it was created. It must remain in the burning forge for fifty years before it finally succumbs to the fire and is consumed.</p>
+			<div class="reference">
+				<div element="ID_WOTC_DMG_ARTIFACT_TABLE" />
+			</div>
+		</description>
+		<setters>
+			<set name="keywords"></set>
+			<set name="category">Magic Weapons</set>
+			<set name="type" addition="battleaxe">Weapon</set>
+			<set name="rarity">Artifact</set>
+			<set name="enhancement">3</set>
+			<set name="attunement">true</set>
+			<set name="weapon">Battleaxe</set>
+		</setters>
+		<rules>
+			<grant type="Vision" id="ID_VISION_DARKVISION" requirements="ID_SRD_RACE_DWARF"/>
+			<grant type="Condition" id="ID_INTERNAL_CONDITION_DAMAGE_IMMUNITY_POISON" requirements="ID_SRD_RACE_DWARF"/>
```

Comment:
As a side note, we need to make it so that if you have resistance and immunity to something then the resistance does not show up